lib/adapters/redis.js

```diff
@@ -3,36 +3,36 @@ const debug = require('debug')('feathers-sync:redis');
 const core = require('../core');
 
 module.exports = config => {
-  return app => {
-    const db = config.uri || config.db;
-    const pub = redis.createClient(db);
-    const sub = redis.createClient(db);
-    const key = config.key || 'feathers-sync';
+    return app => {
+        const key = config.key || 'feathers-sync';
+        const db = config.uri || config.db;
+        if (typeof db !== "undefined")
+            debug(`Setting up Redis client for ${db}`);
+        const pub = config.pub || redis.createClient(db);
```

Comment:
@thavlik I'm not sure if we need to pass two separate Redis clients for pub and sub. What do you think about just accepting a `redisClient` and assigning it to both pub and sub?